programming/sublime-text/pspec.xml

```diff
@@ -5,15 +5,15 @@
         <Name>sublime-text</Name>
         <Homepage>https://www.sublimetext.com/</Homepage>
         <Packager>
-            <Name>Joshua Strobl</Name>
-            <Email>joshua@stroblindustries.com</Email>
+            <Name>Ajang Supardi</Name>
```

Comment:
We don't usually change the "Packager" fields. The script that is used to update these package recipes (see README) also leaves them as is.